docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,231 @@
+.. task:: install_cli ubuntu-14.04
+   :prompt: you@laptop:~$
+
+To test your installation, run the following to check that you have the Flocker client installed correctly:
+
+.. prompt:: bash [you@laptop:~$]
+
+   flocker-deploy --version
+
+Successful installation will display the version of Flocker.
+
+Step 2: Installing Flocker on your hosts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Option A: Installing Flocker on local VMs
+*****************************************
+
+.. note:: You must have `Vagrant <https://www.vagrantup.com/>`_ and `VirtualBox <https://www.virtualbox.org/>`_  installed to create the VMs and start the containers for this tutorial.
+
+In Step 1 you installed the Flocker client on your local machine.
+For the next step in this tutorial you now need two instances of Flocker, each on a separate host.
+Flocker manages the links, ports, and volumes associated with Docker containers and can move them around after deployment.
+To install Flocker (plus dependencies) on the hosts, run the following command and Vagrant will create the environments you need:
+
+.. version-code-block:: console
+
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/Vagrantfile && \
+   curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/cluster.crt && \
+   curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/user.crt && \
+   curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/user.key && \
+   vagrant up && \
+   [ -e "${SSH_AUTH_SOCK}" ] || eval $(ssh-agent) && \
+   ssh-add ~/.vagrant.d/insecure_private_key
+
+Option B: Installing Flocker on AWS or Rackspace
+************************************************
+
+The two instances of Flocker each run on a separate host.
+Flocker manages the links, ports, and volumes associated with Docker containers and can move them around after deployment.
+To install Flocker (plus dependencies), follow the links to the direct instructions:
+
+- :ref:`AWS install instructions <aws-install>`
+- :ref:`Rackspace install instructions <rackspace-install>`
+
+Step 3: Deploying an app on the first host
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You will now have the Flocker client installed on your local machine and two instances of Flocker, each on a different host.
+Now you will create two Docker containers on one of the hosts.
+One is a Python web application and the other is Redis server, which stores its data on a volume.
+
+Download the first 2 .yml files that we have provided:
+
+.. version-code-block:: console
+
+	you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/docker-compose.yml
+	you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/deployment-node1.yml
+
+.. note:: There are 3 .yml files to download. These contain the application and deployment configuration. You can edit these files if you need to change the IP addresses to match your hosts'.
```

Comment:
This extra complexity created by allowing multiple node types is just not worth it in my opinion. It makes a quick tutorial too complex. I would *really* like this to be user tested before merging. We've made the mistake before of adding instructions which haven't been through user testing. I understand that this guide is a product of the user testing for the GSG on Vagrant, but not with custom hosts, right? My intuition is that this is a really hard thing to do and we should wait until we have a provisioning tool or at least positive user testing results before we add this option. At the very least I recommend you try to do it yourself (and time it?).